--- qfaktury/share/applications/qfaktury.desktop	2007-03-20 00:47:05.000000000 +0000
+++ qfaktury.orig/share/applications/qfaktury.desktop	2007-03-20 00:47:51.000000000 +0000
@@ -5,10 +5,9 @@
 Type=Application
 InitialPreference=2
 Categories=Application;Office;X-Red-Hat-Base;X-SuSE-Core-Office;
-Exec=/usr/bin/qfaktury
+Exec=qfaktury
 Name[pl_PL]=QFaktury_GPL
-GenericName[pl_PL]=qfaktury
-Icon=/usr/share/qfaktury/icons/qfaktury_48.png
+Icon=qfaktury.png
 Comment[pl_PL]=Program do drukowania QFaktury_GPL
 Name=QFaktury_GPL
 StartupNotify=true
